devices/iluminize/led_controller_multi_5.dim.json

```diff
@@ -0,0 +1,87 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "iluminize",
+  "modelid": "5112.80",
+  "product": "LED Controller Multi 5 (DIM mode)",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_DIMMABLE_LIGHT",
```

Comment:
That type shouldn’t expose any colour: no `ct`, no `xy`, no `hue`/`sat` not `effect`.

Reply:
You're right. Gonna fix that one (and wonder how candle / fire effects in non-color controllers would look like)